public/index.html

```diff
@@ -9,6 +9,17 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/favicon.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>NEAT+ - Nexus Environment Assessment Tool</title>
+    <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','%REACT_APP_GTM_ID%');</script>
+    <!-- End Google Tag Manager -->
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src=https://www.googletagmanager.com/ns.html?id=%REACT_APP_GTM_ID%
```

Comment:
Missing quotes on src="..." attribute?

Reply:
fixed!!

Reply:
Shouldn't the quotes only contain the link? These attributes shouldn't be within quotes.
`height="0" width="0" style="display:none;visibility:hidden"`